app/scripts/models/user.js

```diff
@@ -27,8 +29,14 @@ define([
       this._marketingEmailClient = options.marketingEmailClient;
       this._assertion = options.assertion;
       this._storage = options.storage || Storage.factory();
+
+      // For now, the uuid is passed in from app-start instead of
+      // being initialized from the resume token or localStorage.
```

Comment:
I'm glad you called this out, because having the app-start.js in charge of loading but this model in charge of storing seems weirdly asymmetrical compared to the way the relier's data is handled.  I can live with it though.